source/fem-dem/vans_assemblers.cc

```diff
@@ -1345,7 +1329,7 @@ VANSAssemblerGidaspow<dim>::calculate_particle_fluid_interactions(
             (18 * pow(cell_void_fraction, -3.65) *
              (1 + 0.15 * pow(Re_p[particle_number], 0.687))) *
             (particle_properties[DEM::CFDDEMProperties::PropertiesIndex::mass] *
-             kinematic_viscosity /
+             kinematic_viscosity[particle_number] /
```

Comment:
I'm not sure i'm a fan of this particle_number, I would rather it be i_particle or something similar, since it is not the number of particles and I find particle_number to be slightly confusing